### DISCUSSION TITLE
gnutls_handshake failed: A TLS fatal alert

### DISCUSSION THREAD

**Discussion by Amachako** [cockpit-project/cockpit#16142] [Q&A] [answered]
Any idea how to solve this?

`cockpit-tls[1259]: cockpit-tls: gnutls_handshake failed: A TLS fatal alert has been received.`

 I can connect only when I'm using the nearest Mesh Wifi of the server. Is it connection timeout related ?

How may I solve this?

**Comment by martinpitt** [comment, marked as answer]
See https://github.com/cockpit-project/cockpit/issues/14896 . In 99% of cases this is just the browser's initial self-signed certificate warning. In the remaining few you may have put cockpit behind a transparent proxy and mis-configured something.
